ios/Classes/BetterPlayer.m

```diff
@@ -846,7 +846,6 @@ - (void) initLimitedBlackCoverView {
 }
 
 - (void) showLimitedBlackCoverView {
-    if (self._betterPlayerView.subviews.contains(_limitedBlackCoverView)) return;
```

Comment:
After re-test, I think we don't need it like other functions: https://github.com/dwango-nfc/betterplayer/blob/23c2c00abc0f4fde272a8bc00a967e0e0c8a645f/ios/Classes/BetterPlayer.m#L906, https://github.com/dwango-nfc/betterplayer/blob/23c2c00abc0f4fde272a8bc00a967e0e0c8a645f/ios/Classes/BetterPlayer.m#L934